todo_app/db/migrate/20170223162711_create_todo.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTodo < ActiveRecord::Migration[5.0]
+  def change
+    create_table :todos do |t|
+      t.string :content
+      t.boolean :is_completed
+
+
+      t.timestamps :updated_at
```

Comment:
You can just say `t.timestamps` and it will take care of `created_at` and `updated_at` for you.